### PR TITLE
Fear/challenge 03 loops conditionals

### PR DESCRIPTION
**Pull request by AGUACATE2304**

### CODE REVIEW

**Comment by AGUACATE2304** [comment]
All the code is formated, at the end of the script line, by the autoSave, I did the  "npx prettier . --write" and didn't refactorized anything

File: challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+
+console.log("---------Conditionals Exercice_01---------");
+
+let randomNumber = 70;
+
+if (randomNumber % 2 === 0) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js]: ⚠️ FIX: Change the variable to test it works for different cases. Print result.

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
+        1. Create a for loop that will print out all the even numbers between 10 and 40.
+        
+        2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+console.log("---------loops Exercice_00---------");
+
+for (let number = 10; 40 >= number; number++) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js]:
```suggestion
for (let number = 10; number <= 40 ; number++) {
```

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  if (number % 2 === 0) {
+    console.log(number + " is even");
+  }
+
+  if (number % 2 === 1) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js]:
```suggestion
  if (number % 2 !== 0) {
```

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+console.log("---------while Exercice_01---------");
+
+let number = 10;
+console.log("Even numbers between 10 & 40:");
+while (number <= 40) {
+  if (number % 2 === 0) {
+    console.log(number);
+  }
+  number++;
+  if (number > 40) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js]: This check is not necessary. While will stop execution when its condition is evaluated as `true`.

⚠️ FIX: remove final check

File: challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+number = 10;
+console.log("Odd numbers between 10 & 40:");
+while (number <= 40) {
+  if (number % 2 === 1) {
+    console.log(number);
+  }
+  number++;
+  if (number > 40) {
```
Comment (tictools) [inline challenge_03_loops-conditionals/loops-conditionals/while/exercise_01.js]: This check is not necessary. While will stop execution when its condition is evaluated as `true`.

⚠️ FIX: remove final check

**Review by tictools — CHANGES_REQUESTED** [review]

File: challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 6]
```diff
@@ -32,10 +32,10 @@ const isOddLoop = function (lowThreshold, highThreshold) {
 console.log("###############################");
 console.log("--- Is EVEN or ODD loop Program ---");
 
-const startAt = 10;
```
Comment (MartiCobos99) [inline challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js]: FIX! You inicialize the function without value's

NEED WORKS🤔

**Review by MartiCobos99 — COMMENTED** [review]
KEEP WORKING ON IT 🤔

File: challenge_03_loops-conditionals/README.md
Comment (tictools) [inline challenge_03_loops-conditionals/README.md:1]: FIX: remove this file. README should live only at project root level.

Comment (tictools) [inline challenge_03_loops-conditionals/README.md:1]: FIX: move this file. README should live only at project root level.

**Review by tictools — CHANGES_REQUESTED** [review]

**Review by MartiCobos99 — APPROVED** [review]
All in good conditions.

**Review by tictools — APPROVED** [review]

**Pull request merged**